python/sglang/srt/layers/amx_utils.py

```diff
@@ -6,6 +6,15 @@
 
 logger = logging.getLogger(__name__)
 
+from enum import IntEnum
+
+
+class CPUMoECompMethod(IntEnum):
+    BF16_GEMM = 0
+    INT8_W8A8_GEMM = 1
+    FP8_W8A16_GEMM = 2
+    INT4_W4A8_GEMM = 3
```

Comment:
previously i was just following the triton implementation APIs, it is time to simplify this a littble.

since we are not likely to provide a second implementation for a particular data type, just remove this as well `ues_int8_w8a8`, `ues_fp8_w8a16`.

that is to say, we can check weight data type and int8 just means it8_wa8a8, and so on.

and also remove `a1_scale` and `a2_scale`, we are not likely to support static quantization of activations.